src/protocol/profiler.rs

```diff
@@ -0,0 +1,87 @@
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+/// Coverage data for a source range.
+pub struct CoverageRange {
+    pub start_offset: u16,
+    pub end_offset: u16,
+    pub count: u16,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+/// Coverage data for a JavaScript function.
+pub struct FunctionCoverage {
+    pub function_name: String,
+    /// Source ranges inside the function with coverage data.
+    pub ranges: Vec<CoverageRange>,
+}
+
+/// JS line coverage information for a single script
+/// See https://chromedevtools.github.io/devtools-protocol/tot/Profiler#type-ScriptCoverage
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ScriptCoverage {
+    pub script_id: String,
+    /// Either the name or URL of a script loaded by the page
+    pub url: String,
+    /// Functions contained in the script that has coverage data
```

Comment:
`have`

Reply:
This is copied from the official docs, and I'm guessing the 'has' is referring to the script rather than the functions?